stargate/crates/stargate-lib/src/common.rs

```diff
@@ -27,6 +27,10 @@ pub struct Opt {
     /// Accepts [http|udp]://host:port/path (path is optional)
     #[structopt(short, long)]
     pub tracing_endpoint: Option<TracingConfig>,
+
+    /// A space separated list of header names which Stargate should propagate to downstream services.
```

Comment:
Heh [this is my one of my most popular tweets](https://twitter.com/rmgn/status/650069074769149952), and sort of shows that the use of downstream/upstream is not ideal. Let's use `implementing services` if that's our federation nomenclature

Reply:
I agree we should pick one nomenclature, but it's worth pointing out that we call it [`downstream service(s)` in a lot of places](https://github.com/apollographql/federation/search?q=%22downstream+service%22+OR+%22downstream+services%22)! ([vs. `implementing service(s)`](https://github.com/apollographql/federation/search?q=%22implementing+service%22+OR+%22implementing+services%22)).

Since this is just a comment, this might be a better suggestion for a wholesale name re-alignment.  We should probably have that conversation with product and do it the change at once more broadly though and some of us have recently noted that _implementing services_ is a name that might benefit from revisiting.

Reply:
Implementing is a good suggestion for now. I hope we do have that conversation in the future, I'm rooting for `subgraph` :)